app/javascript/techradar.js

```diff
@@ -20,11 +23,14 @@
             var height = (radius * _.size(this.radar.arcs)) * 2 + (offset * 2);
             var center = {x: width / 2, y: height / 2};
 
+            //noinspection HtmlUnknownTarget
             var linkTemplate = _.template("<a target='_blank' href='${url}'>${title}</a>");
 
+            var extraSpacing = 20; //Extra spacing on the right for texts in tooltip
```

Comment:
This is a brittle solution which does not work in case a spot is actually placed near the right border of a radar